### ISSUE TITLE
Settings django admin header

### ISSUE THREAD

**Issue by RCheese** [fabiocaccamo/django-admin-interface#75]
How can I do this? It doesn't work with admin-interface module
```python
admin.site.site_header = f'DJANGO ADMIN. ref:{settings.COMMIT} version:{settings.TAG}'
```

**Comment by fabiocaccamo** [comment]
You can customize the admin title using the theme model-admin, but it doesn't support what you need. Feel free to submit a PR.

**Comment by fabiocaccamo** [comment]
Fixed in `0.12.1` version.
